src/Commands/RunScan.spec.ts

```diff
@@ -67,4 +88,123 @@ describe('RunScan', () => {
       expect(act).toThrowError(SyntaxError);
     });
   });
+
+  describe('handler', () => {
+    let runScan: RunScan;
+    let mockedRestScans: RestScans;
+
+    beforeEach(() => {
+      mockedRestScans = mock(RestScans);
+      container.registerInstance(Scans, instance(mockedRestScans));
+      resetCalls(mockedRestScans);
+      runScan = new RunScan();
+    });
+
+    afterEach(() => {
+      container.clearInstances();
+      reset(mockedRestScans);
+    });
+
+    it('should correctly pass scan config from args', async () => {
+      const args = {
```

Comment:
please put AAA comments in all test cases
```suggestion
      // arrange
      const args = {
```

Reply:
added everywhere